plugins/catalog/src/index.ts

```diff
@@ -40,3 +40,6 @@ export {
   EntityLinksCard,
   EntitySystemDiagramCard,
 } from './plugin';
+export type { EntityRow } from './components/CatalogTable/types';
```

Comment:
This is now exported as of https://github.com/backstage/backstage/pull/5984, so be sure to align with that